src/app/dev/DevToys.Api/Tool/GUI/Components/UIHoverTooltip.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace DevToys.Api;
+
+public record UIHoverTooltip
```

Comment:
Please add XML documentation for all the public parts here. It will be exposed in the documentation.